Model/main.py

```diff
@@ -77,11 +77,17 @@
         float(INVALID_SPEND_RATE),
         float(CATASTROPHE_RATE),
         float(DELEGATE_RATE),
+        with_scale_fixed=True,
         with_balance=True,
-        # with_fb_coins_dist=True,
-        with_cum_op_cost=True,
-        with_divergence=True,
-        with_overpayments=True,
+        with_divergence=False,
+        with_op_cost=False,
+        with_cum_op_cost=False,
+        with_overpayments=False,
+        with_coin_pool=True,
+        with_coin_pool_age=True,
+        with_risk_status=False,
+        with_risk_time=False,
+        with_fb_coins_dist=True,
```

Comment:
Can you make those env vars too please? (and have a separate table for those optional vars in the README)